cli/damas

```diff
@@ -271,10 +279,10 @@ case $COMMAND in
       printf "\n\n"
     fi
     RES=$(eval "curl $CURL_VERBOSE -ks -w \"\n%{http_code}\" --fail -d 'username=$USERN&password=${PASS}${expiresIn}' ${DAMAS_SERVER}/api/signIn/")
-    TOKEN=$(echo $RES| sed 's/^.*"token":"\([^"]*\)".*$/\1/')
+    TOKEN=$(echo $RES | sed 's/^.*"token":"\([^"]*\)".*$/\1/')
+    DAMAS_TOKEN=$TOKEN
     TOKEN_FILE="/tmp/damas-$USER-$(echo -n $DAMAS_SERVER | cksum | head -c 10)"
-    echo $TOKEN
```

Comment:
I think it's better since the user does not actually need the token if he's already logged in. Otherwise if he wants one, he can generate it with the view provided.